### PR TITLE
Add module information to README

### LINKED ISSUE #31: Install instructions missing a step
I tried setting up the play-2.4 version on a Play Java app and it failed with a NullPointerException.

The reason is the docs are missing the instructions to add `play.modules.enabled += "play.modules.swagger.SwaggerModule"` to `application.conf`.

### PR DESCRIPTION
**Pull request by josephearl**
Fixes #31. Also fixes the step numbering.

### CODE REVIEW

File: play-2.4/swagger-play2/README.md [hunk 1]
```diff
@@ -44,7 +44,13 @@ sbt publishLocal
...
 There are just a couple steps to integrate your Play2 app with swagger.
 
-1.  Add the resource listing to your routes file (you can read more about the resource listing [here](https://github.com/swagger-api/swagger-core/wiki/Resource-Listing)
+1\. Add the Swagger module to your `application.conf`
+ 
+```
+play.modules.enabled += "play.modules.swagger.SwaggerModule"
+```
+ 
+2\. Add the resource listing to your routes file (you can read more about the resource listing [here](https://github.com/swagger-api/swagger-core/wiki/Resource-Listing)
```
Comment (benmccann) [inline play-2.4/swagger-play2/README.md]: i think you're missing a close paren

Comment (josephearl) [inline play-2.4/swagger-play2/README.md]: Fixed

**Comment by benmccann** [comment]
Do you mind squashing the commits?

@fehguy this one looks good to me

**Comment by josephearl** [comment]
Squashed

**Pull request merged**